src/main/java/com/whiletrue/tododemo/dto/TaskResponse.java

```diff
@@ -18,14 +19,14 @@ public class TaskResponse {
     private LocalDateTime created;
     private LocalDateTime updated;
 
-    public TaskResponse(Long id, Task task) {
-        this.id = id;
-        this.name = task.getName();
-        this.description = task.getDescription();
-        this.dueDateTime = task.getDueDateTime();
-        this.createdBy = task.getCreatedBy();
-        this.assignedTo = task.getAssignedTo();
-        this.completed = task.isCompleted();
+    public TaskResponse(TaskEntity taskEntity) {
```

Comment:
Ussually we name it just "Task" instead of "TaskEntity" .

Reply:
I use `Task` as DTO. Should I change it to `TaskRequest` or `TaskDto`?

Reply:
yup